lib/LedgerSMB/Scripts/import_csv.pm

```diff
@@ -450,7 +451,8 @@ sub _process_parts {
                                               AND al.description = ? ))'
                   : ($_ eq 'partsgroup' ?
                      '(SELECT id FROM partsgroup WHERE partsgroup = ?)'
-                     : $_))}
+        #LZY             : $_))}
```

Comment:
Please don't leave the old (faulty) line in place: we have git to help us track the history of our changes (and we can always find back who changed what). In this case, it also made it harder to see the actual difference.

Reply:
yes, will remove the old line